### ISSUE TITLE
Unable to upgrade package

### ISSUE THREAD

**Issue by dvzrv** [surge-synthesizer/stochas#141]
Hi! :wave: 

I'm packaging this project for Arch Linux. Since some time around 1.3.5 I am unable to upgrade the package.
The bundled juce version in 1.3.9 is not compatible with gcc 13.2.1 and although I have up until now used a devendored system-wide provided juce (currently 7.0.10) and that works with 1.3.5, it does not work with 1.3.9.

Some time after 1.3.5 the clap plugin integration has been added and this only recently got C++17 support (the vendored version only has 14 support).

Long story short: Can you please upgrade to a more recent juce and clap integration and create a release? :)

**Comment by baconpaul** [comment]
Thabks! Will take a look 

but the old juce problem is just a utility include right? You might be able to patch very quickly with just a -Iutility to the compile commands and get in the water earlier

**Comment by dvzrv** [comment]
with both gcc 13.2.1 and 12.3.0 I get

```
  In file included from
  /build/stochas/src/stochas/lib/JUCE/modules/juce_gui_basics/juce_gui_basics.h:271:



  /build/stochas/src/stochas/lib/JUCE/modules/juce_gui_basics/windows/juce_ComponentPeer.h:
  In member function ‘void juce::ComponentPeer::setAppStyle(Style)’:


  /build/stochas/src/stochas/lib/JUCE/modules/juce_gui_basics/windows/juce_ComponentPeer.h:504:18:
  error: ‘exchange’ is not a member of ‘std’

    504 |         if (std::exchange (style, s) != style)
        |                  ^~~~~~~~

  make[2]: *** [extras/Build/juceaide/CMakeFiles/juceaide.dir/build.make:76:
  extras/Build/juceaide/CMakeFiles/juceaide.dir/Main.cpp.o] Error 1

  make[1]: *** [CMakeFiles/Makefile2:131:
  extras/Build/juceaide/CMakeFiles/juceaide.dir/all] Error 2

  make: *** [Makefile:136: all] Error 2
```

**Comment by dvzrv** [comment]
Hm, I'd probably better wait until there's a newer release. This takes too much time to patch otherwise. :sweat_smile:

**Comment by baconpaul** [comment]
Yeah that's because the juce we had doesn't include <utility> which is why the cmake patch will work

but i just tried it with J710/C120 and it seemed to work. I'll toss in a PR and see what @rudeog things.

Can you build from a branch to test integration?

**Comment by baconpaul** [comment]
Linking to #142

**Comment by baconpaul** [comment]
looks like we got tagged as 1.3.10 just now! Wanna give it a whirl @dvzrv ?

**Comment by dvzrv** [comment]
Thanks a lot! Was just able to build 1.3.10 and upgrade! :tada: 

https://gitlab.archlinux.org/archlinux/packaging/packages/stochas/-/commits/main
